platform/android/java/lib/src/org/godotengine/godot/input/GodotEditText.java

```diff
@@ -101,7 +101,7 @@ protected void initView() {
 	}
 
 	public boolean isMultiline() {
-		return mMultiline;
+		return mInputType == GodotIO.KEYBOARD_TYPE_MULTILINE;
 	}
```

Comment:
We should probably add another `getKeyboardType()` method to complement the added functionality.

If we got that way, we should replace the `int` constants with java enums so the `getKeyboardType()` method returns an unambiguous value.

Reply:
> If this is something that's needed in the future, sure.

The publicly exposed methods are used by Godot Android plugins, and/or projects that use the engine as a library. So if we're expanding the set of supported keyboard options, we need to provide a mechanism to query the selected keyboard type, hence the addition of the new method.

> this would require adding a redundant int->enum mapping for a privately used member.

Not sure I understand what you mean by `redundant int->enum mapping`.
You can either declare the enum with the same mapping as the native version which will allow to use the int value as index into the java enum values.
Or you can add a private `int` field to the declared enum class, and add logic to perform the mapping.

Both approaches are fairly common in native/java projects and should be straightforward.